### ISSUE TITLE
Collada/Gltf Export: Missing UV-Coords

### ISSUE THREAD

**Issue by Oribow** [3dcitydb/importer-exporter#35]
The Importer/Exporter exports GLTF-models without texture coordinates.
Apparently, the Collada files a malformed, so the collada2gltf tools ignores the texture coords. 

Example:
```
<effect id="_tex4666_eff">
            <profile_COMMON>
                <newparam sid="_tex4666_surface">
                    <surface type="2D">
                        <init_from>_tex4666_img</init_from>
                    </surface>
                </newparam>
                <newparam sid="_tex4666_sampler">
                    <sampler2D>
                        <source>_tex4666_surface</source>
                    </sampler2D>
                </newparam>
                <technique sid="COMMON">
                    <lambert>
                        <emission>
                            <color>0.0 0.0 0.0 1.0</color>
                        </emission>
                        <diffuse>
                            <texture texture="_tex4666_sampler" texcoord="TEXCOORD"/>
                        </diffuse>
                    </lambert>
                </technique>
                <extra>
                    <technique profile="GOOGLEEARTH">
                        <double_sided>0</double_sided>
                    </technique>
                </extra>
            </profile_COMMON>
        </effect>
```
According to the schema specs (https://www.khronos.org/files/collada_spec_1_4.pdf, site 225), when `texcoord="TEXCOORD"` is used, there also needs to be a line like this later on:

`< bind_vertex _input semantic=" TEXCOORD" input_semantic=" TEXCOORD " /> `

But no such line exists in the generated Collada-file.. This causes the collada2gltf tool to just drop the TexCoord attribute in the GLTF-file. To fix this, either remove the `texcoord="TEXCOORD"` attribute or add the bind_vertex node in.

**Comment by yaozhihang** [comment]
According to the collada_spec_1.4, the `<bind_vertex _input>` element ist optional (minOccurs = 0). Also, In the collada file generated from our import-export tool, there is also no mismatches in the semantic identifier’s meaning, which requires the  `<bind_vertex _input>` element. 

A short quesion: Has the import-export tool converted your textured 3D city models to glTF-models with missing texutre information which cannot  be properly visualized on Cesium?  If so, could you share with us your CityGML data which would be very helpful in fixing the bug?

**Comment by Oribow** [comment]
All of the CityGML-data I used comes from this site: http://www.businesslocationcenter.de/berlin3d-downloadportal/.

I don't know Collada's Schema very well, and the above was only my attempt at explaining this problem. I came to this conclusion, after comparing the collada file from the exporter, with the same file, but imported into Blender, then exported as Collada again. Only the "Blender-file" could be converted correctly with the Collada2Gltf-tool.

Since both Blender and Google-Earth-Pro had no problem reading the Collada-file from the exporter, maybe the Collada2Gltf-tool is at fault here.
